client/src/transmitter/index.ts

```diff
@@ -327,8 +344,29 @@ function setup_media_recorder_listener(): void {
   }
 }
 
-function plunk_media_recorder_listener(): void {
-  media_recorder.ondataavailable = null
+const MEDIA_RECORDER_T = 2000
+
+let media_recorder_started = false
+
+function start_media_recorder(): void {
+  if (media_recorder_started) {
+    return
+  }
+  console.log('start_media_recorder')
+  media_recorder_started = true
+  media_recorder.start(MEDIA_RECORDER_T)
+}
+
+function stop_media_recorder(): void {
+  if (!media_recorder_started) {
+    return
+  }
+  if (media_recorder !== 'recording') {
+    return
+  }
```

Comment:
```suggestion
  if (!media_recorder_started || media_recorder !== 'recording') {
    return
  }
```

Reply:
It would be hard to know in a debugging session which one of the conditions were true.